ui/app/templates/components/mfa/mfa-form.hbs

```diff
@@ -3,74 +3,73 @@
   SPDX-License-Identifier: BUSL-1.1
 }}
 
-<div class="auth-form" data-test-mfa-form>
-  <div class="box is-marginless is-shadowless">
-    <p data-test-mfa-description>
-      {{this.description}}
-    </p>
-    <form id="auth-form" {{on "submit" this.submit}}>
-      <MessageError @errorMessage={{this.error}} />
-      <div class="field has-top-margin-l">
-        {{#each this.constraints as |constraint index|}}
-          {{#if index}}
-            <hr />
-          {{/if}}
-          {{#if (gt constraint.methods.length 1)}}
-            <Select
-              @label="Multi-factor authentication method"
-              @options={{constraint.methods}}
-              @valueAttribute={{"id"}}
-              @labelAttribute={{"label"}}
-              @isFullwidth={{true}}
-              @noDefault={{true}}
-              @selectedValue={{constraint.selectedId}}
-              @onChange={{fn this.onSelect constraint}}
-              data-test-mfa-select={{index}}
+<Hds::Card::Container @level="high" @hasBorder={{true}} class="has-padding-l" data-test-mfa-form>
```

Comment:
Misleading diff because I just wrapped the MFA component in `Hds::Card`. To view changes change to ✔️ hide whitespace, only other change is moved the `Back` button here

<img width="890" alt="Screenshot 2025-04-30 at 11 32 45 AM" src="https://github.com/user-attachments/assets/bfc99f94-24a5-418f-bb38-75c6b6456b29" />

Reply:
Nice thanks for the screenshots to diff!